card/src/main/java/com/adyen/checkout/card/internal/ui/view/InstallmentListAdapter.kt

```diff
@@ -66,8 +68,11 @@ internal class InstallmentListAdapter(
 
 internal data class InstallmentModel(
     @StringRes val textResId: Int,
-    val value: Int?,
-    val option: InstallmentOption
+    val monthValue: Int?,
```

Comment:
Good idea renaming this, makes it clearer 👍 However I still had to look it up to understand exactly what it means. The [documentation in the API explorer](https://docs.adyen.com/api-explorer/Checkout/71/post/payments#request-installments-value) says this:
> Defines the number of installments.

Maybe it can help us pick a better name?